keyboards/ZTBoards/after/after.h

```diff
@@ -0,0 +1,23 @@
+#ifndef AFTER_H
+#define AFTER_H
+
+#include "quantum.h"
+//#define XXX KC_NO
+//include the above later to optimize after.h file
+//lets me use xxx instead of kc_no
+
```

Comment:
```suggestion
#pragma once

#include "quantum.h"

```

Reply:
Do I just need to hit commit suggestion for these or do I need to go back and change it on my end?